Infrastructure/Models/Parcel.cs

```diff
@@ -7,11 +7,22 @@
 
 namespace Infrastructure.Models
 {
+    public enum Size
+    {
+        S, M, L, XL
+    }
```

Comment:
I think it would be better to create enum in another file, rather than having everything on the same. [Possible argument](https://stackoverflow.com/a/2283094)